sdk/playwrighttesting/microsoft-playwright-testing/CHANGELOG.md

```diff
@@ -1,15 +1,23 @@
 # Release History
 
-## 1.0.0-beta.5 (Unreleased)
+## 1.0.0-beta.5 (2024-11-15)
+
+This release fixes a bug related to upload method and few Engineering Systems updates.
 
 ### Features Added
 
 ### Breaking Changes
 
 ### Bugs Fixed
 
+-Fixed a bug where the upload buffer and upload file methods were not awaited, which sometimes resulted in 404 errors in the web application.
+
 ### Other Changes
 
+-Removed runName support from the wsEndpoint as auth service needs to handle URL encoding properly.
+
+-Added X-MS-Package-Version header to connect session API for language and package version decoding.
```

Comment:
-Added X-MS-Package-Version header to cloud hosted browser API for language and package version decoding.